app/utils/drivers.win32.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2015-2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import _ from 'lodash';
+//import regedit from 'regedit';
```

Comment:
There hasn't been significant testing on either option, so I wasn't comfortable dropping the code yet. Once we get into some actual alpha testing and confidence in the currently used package/method is bolstered it certainly can be dropped. I'll put a TODO here.